test/e2e/case28_evauluation_metric_test.go

```diff
@@ -0,0 +1,122 @@
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+var _ = Describe("Test config policy evaluation metrics", Ordered, func() {
+	const (
+		policyName = "policy-cfgmap-watch"
+		policyYaml = "../resources/case28_evaluation_metric/case28_policy.yaml"
+	)
+
+	prePolicyEvalDuration := -1
+
+	It("should record number of ", func() {
+		evalMetric := utils.GetMetrics(
+			"config_policies_evaluation_duration_seconds_bucket", fmt.Sprintf(`le=\"%d\"`, 30))
+		Expect(len(evalMetric) != 0)
```

Comment:
There are a few assertions like this, but I'm not sure if they work. I think they need `.To(...)` or `.NotTo(...)` in order to actually do something. So this could have either:
```
Expect(len(evalMetric) != 0).To(BeTrue())
// or
Expect(evalMetric).NotTo(BeEmpty())
```